README.md

```diff
@@ -1 +1,72 @@
-# Aqovia.HttpClient.CacheProvider
\ No newline at end of file
+# README #
+
+## Aqovia.CachingHttpClient
+
+Nuget Info
+
+[![NuGet Info](https://buildstats.info/nuget/Aqovia.CachingHttpClient?includePreReleases=true)](https://www.nuget.org/packages/Aqovia.CachingHttpClient/)
+
+Build status
+
+[![.NET Core](https://github.com/Aqovia/Aqovia.CachingHttpClient/workflows/Nuget%20Publish%20CI/badge.svg?branch=master)](https://github.com/Aqovia/Aqovia.CachingHttpClient/actions?query=branch%3Amaster)
+
+[![Windows Build history](https://buildstats.info/github/chart/Aqovia/Aqovia.CachingHttpClient?branch=master&includeBuildsFromPullRequest=false)](https://github.com/Aqovia/Aqovia.CachingHttpClient/actions?query=branch%3Amaster)
+
+## About
+
+
+## Getting started
+
+* Include the Nuget package in your test project
+    - https://www.nuget.org/packages/Aqovia.CachingHttpClient/
+* Either use the CachingHelper to create a CachingHttpClient or use ASP.NET Core DI to get HttpClient
+
+
+## Project overview
+
+The project contains the following directory structure
+
+```
+src/
+    Aqovia.CachingHttpClient
+    Aqovia.CachingHttpClient.AspNetCoreApi
+test/
+    Aqovia.CachingHttpClient.Tests
+```
+
+### src/Aqovia.CachingHttpClient
+
+Contains the source library code providing a helper class to create ICacheStore and HttpClient with a default CacheStore.
+
+### src/Aqovia.CachingHttpClient.AspNetCoreApi

```

Comment:
If it is an example project, I'd move this to "examples" folder so that it is not the part of the source code.

Reply:
moved to examples folder